packages/playground/src/components/branches-list.tsx

```diff
@@ -57,15 +98,21 @@ export default function BranchesList({ branches }: PageProps) {
                         {branch.name}
                       </Button>
                     </Text>
-                    <Button variant="ghost" size="xs">
-                      <Icon name="clone" size={16} className="text-tertiary-background" />
-                    </Button>
+                    {CopyButton()}
```

Comment:
Let's render `CopyButton` as a component instead `<CopyButton />`.

Reply:
Migrate it as a component in the playground, hold on this copy first since there're lots of place using copied function and waiting for the final design